{"level":"error","message":"ReferenceError: passmword is not definedReferenceError: passmword is not defined","stack":"Error: ReferenceError: passmword is not defined\n    at Object.login (C:\\Projects\\ex_team\\services\\modules\\error-handler.js:6:19)"}
{"level":"error","message":"ReferenceError: passmword is not definedReferenceError: passmword is not defined","stack":"Error: ReferenceError: passmword is not defined\n    at Object.login (C:\\Projects\\ex_team\\services\\modules\\error-handler.js:6:19)"}
{"level":"error","message":"ReferenceError: passmword is not definedReferenceError: passmword is not defined","stack":"Error: ReferenceError: passmword is not defined\n    at Object.login (C:\\Projects\\ex_team\\services\\modules\\error-handler.js:7:19)"}
{"level":"error","message":"ReferenceError: passmword is not definedReferenceError: passmword is not defined","stack":"Error: ReferenceError: passmword is not defined\n    at Object.login (C:\\Projects\\ex_team\\services\\modules\\error-handler.js:7:19)"}
{"level":"error","message":"Something went wrong. Please try again later.Something went wrong. Please try again later.","stack":"Error: Something went wrong. Please try again later.\n    at Object.login (C:\\Projects\\ex_team\\services\\modules\\error-handler.js:9:23)"}
{"level":"error","message":"Something went wrong. Please try again later.Something went wrong. Please try again later.","stack":"Error: Something went wrong. Please try again later.\n    at Object.login (C:\\Projects\\ex_team\\services\\modules\\error-handler.js:10:23)"}
{"level":"error"}
{"level":"error","message":"Something went wrong. Please try again later.Something went wrong. Please try again later.","stack":"Error: Something went wrong. Please try again later.\n    at Object.login (C:\\Projects\\ex_team\\services\\modules\\error-handler.js:11:23)"}
{"level":"error"}
{"level":"error","message":"Something went wrong. Please try again later.Something went wrong. Please try again later.","stack":"Error: Something went wrong. Please try again later.\n    at Object.login (C:\\Projects\\ex_team\\services\\modules\\error-handler.js:11:23)"}
{"level":"error"}
{"level":"error","message":"Something went wrong. Please try again later.Something went wrong. Please try again later.","stack":"Error: Something went wrong. Please try again later.\n    at Object.login (C:\\Projects\\ex_team\\services\\modules\\error-handler.js:11:23)"}
{"level":"error","message":"ReferenceError: passmword is not definedReferenceError: passmword is not defined","stack":"Error: ReferenceError: passmword is not defined\n    at Object.login (C:\\Projects\\ex_team\\services\\modules\\error-handler.js:10:23)"}
{"level":"error","message":"ReferenceError: passmword is not definedReferenceError: passmword is not defined","stack":"Error: ReferenceError: passmword is not defined\n    at Object.login (C:\\Projects\\ex_team\\services\\modules\\error-handler.js:8:23)"}
{"level":"error","message":"ReferenceError: passmword is not definedReferenceError: passmword is not defined","stack":"Error: ReferenceError: passmword is not defined\n    at Object.login (C:\\Projects\\ex_team\\services\\modules\\error-handler.js:8:23)"}
{"level":"error","message":"Invalid UserName or password.Invalid UserName or password.","stack":"Error: Invalid UserName or password.\n    at Object.login (C:\\Projects\\ex_team\\services\\modules\\error-handler.js:6:19)\n    at process._tickCallback (internal/process/next_tick.js:68:7)"}
{"level":"error","message":"Invalid UserName or password.Invalid UserName or password.","stack":"Error: Invalid UserName or password.\n    at Object.login (C:\\Projects\\ex_team\\services\\modules\\error-handler.js:6:19)\n    at process._tickCallback (internal/process/next_tick.js:68:7)"}
{"level":"error","message":"ReferenceError: passwokrd is not definedReferenceError: passwokrd is not defined","stack":"Error: ReferenceError: passwokrd is not defined\n    at Object.login (C:\\Projects\\ex_team\\services\\modules\\error-handler.js:6:19)"}
{"level":"error","message":"Error: Invalid UserName or password.Error: Invalid UserName or password.","stack":"Error: Error: Invalid UserName or password.\n    at Object.login (C:\\Projects\\ex_team\\services\\modules\\error-handler.js:6:19)\n    at process._tickCallback (internal/process/next_tick.js:68:7)"}
{"level":"error","message":"Error: Invalid UserName or password.Error: Invalid UserName or password.","stack":"Error: Error: Invalid UserName or password.\n    at Object.login (C:\\Projects\\ex_team\\services\\modules\\error-handler.js:6:19)\n    at process._tickCallback (internal/process/next_tick.js:68:7)"}
{"level":"error","message":"Invalid UserName or password.Invalid UserName or password.","stack":"Error: Invalid UserName or password.\n    at Object.login (C:\\Projects\\ex_team\\services\\modules\\error-handler.js:6:19)\n    at process._tickCallback (internal/process/next_tick.js:68:7)"}
{"level":"error","message":"Invalid UserName or password.Invalid UserName or password.","stack":"Error: Invalid UserName or password.\n    at Object.login (C:\\Projects\\ex_team\\services\\modules\\error-handler.js:6:19)\n    at process._tickCallback (internal/process/next_tick.js:68:7)"}
{"level":"error","message":"Error: Invalid UserName or password.Error: Invalid UserName or password.","stack":"Error: Error: Invalid UserName or password.\n    at Object.login (C:\\Projects\\ex_team\\services\\modules\\error-handler.js:6:19)\n    at process._tickCallback (internal/process/next_tick.js:68:7)"}
{"level":"error","message":"Error: Invalid UserName or password.Error: Invalid UserName or password.","stack":"Error: Error: Invalid UserName or password.\n    at Object.login (C:\\Projects\\ex_team\\services\\modules\\error-handler.js:6:19)\n    at process._tickCallback (internal/process/next_tick.js:68:7)"}
{"level":"error","message":"Invalid UserName or password.Invalid UserName or password.","stack":"Error: Invalid UserName or password.\n    at Object.login (C:\\Projects\\ex_team\\services\\modules\\error-handler.js:6:19)\n    at process._tickCallback (internal/process/next_tick.js:68:7)"}
{"level":"error","message":"[object Object][object Object]","stack":"Error: [object Object]\n    at Object.login (C:\\Projects\\ex_team\\services\\modules\\error-handler.js:6:19)\n    at process._tickCallback (internal/process/next_tick.js:68:7)"}
{"level":"error","message":"[object Object][object Object]","stack":"Error: [object Object]\n    at Object.login (C:\\Projects\\ex_team\\services\\modules\\error-handler.js:6:19)\n    at process._tickCallback (internal/process/next_tick.js:68:7)"}
{"level":"error","message":"Error: [object Object]Error: [object Object]","stack":"Error: Error: [object Object]\n    at Object.login (C:\\Projects\\ex_team\\services\\modules\\error-handler.js:6:19)\n    at process._tickCallback (internal/process/next_tick.js:68:7)"}
{"level":"error","message":"[object Object][object Object]","stack":"Error: [object Object]\n    at Object.login (C:\\Projects\\ex_team\\services\\modules\\error-handler.js:6:19)\n    at process._tickCallback (internal/process/next_tick.js:68:7)"}
{"level":"error","message":"TypeError: serviceError is not a constructorTypeError: serviceError is not a constructor","stack":"Error: TypeError: serviceError is not a constructor\n    at Object.login (C:\\Projects\\ex_team\\services\\modules\\error-handler.js:6:19)\n    at process._tickCallback (internal/process/next_tick.js:68:7)"}
{"level":"error","message":"TypeError: serviceError is not a constructorTypeError: serviceError is not a constructor","stack":"Error: TypeError: serviceError is not a constructor\n    at Object.login (C:\\Projects\\ex_team\\services\\modules\\error-handler.js:6:19)\n    at process._tickCallback (internal/process/next_tick.js:68:7)"}
{"level":"error","message":"CustomError: bazMessageCustomError: bazMessage","stack":"Error: CustomError: bazMessage\n    at Object.login (C:\\Projects\\ex_team\\services\\modules\\error-handler.js:6:19)\n    at process._tickCallback (internal/process/next_tick.js:68:7)"}
{"level":"error","message":"ServiceError: bazMessageServiceError: bazMessage","stack":"Error: ServiceError: bazMessage\n    at Object.login (C:\\Projects\\ex_team\\services\\modules\\error-handler.js:6:19)\n    at process._tickCallback (internal/process/next_tick.js:68:7)"}
{"level":"error","message":"TypeError: serviceError is not a constructorTypeError: serviceError is not a constructor","stack":"Error: TypeError: serviceError is not a constructor\n    at Object.login (C:\\Projects\\ex_team\\services\\modules\\error-handler.js:6:19)\n    at process._tickCallback (internal/process/next_tick.js:68:7)"}
{"level":"error","message":"TypeError: serviceError is not a constructorTypeError: serviceError is not a constructor","stack":"Error: TypeError: serviceError is not a constructor\n    at Object.login (C:\\Projects\\ex_team\\services\\modules\\error-handler.js:6:19)\n    at process._tickCallback (internal/process/next_tick.js:68:7)"}
{"level":"error","message":"TypeError: serviceError is not a constructorTypeError: serviceError is not a constructor","stack":"Error: TypeError: serviceError is not a constructor\n    at Object.login (C:\\Projects\\ex_team\\services\\modules\\error-handler.js:6:19)\n    at process._tickCallback (internal/process/next_tick.js:68:7)"}
{"level":"error","message":"ServiceError: bazMessageServiceError: bazMessage","stack":"Error: ServiceError: bazMessage\n    at Object.login (C:\\Projects\\ex_team\\services\\modules\\error-handler.js:6:19)\n    at process._tickCallback (internal/process/next_tick.js:68:7)"}
{"name":"ServiceError","level":"error","message":"ServiceError: bazMessageServiceError: bazMessage","stack":"ServiceError: ServiceError: bazMessage\n    at Object.login (C:\\Projects\\ex_team\\services\\modules\\error-handler.js:9:19)\n    at process._tickCallback (internal/process/next_tick.js:68:7)"}
{"name":"ServiceError","level":"error","message":"Error: bazMessageError: bazMessage","stack":"ServiceError: Error: bazMessage\n    at Object.login (C:\\Projects\\ex_team\\services\\modules\\error-handler.js:9:19)\n    at process._tickCallback (internal/process/next_tick.js:68:7)"}
{"name":"ServiceError","date":"2019-08-21T05:55:58.706Z","level":"error","message":"Error: bazMessageError: bazMessage","stack":"ServiceError: Error: bazMessage\n    at Object.login (C:\\Projects\\ex_team\\services\\modules\\error-handler.js:9:19)\n    at process._tickCallback (internal/process/next_tick.js:68:7)"}
{"name":"ServiceError","date":"2019-08-21T06:03:21.049Z","status":500,"level":"error","message":"ServiceError: bazMessageServiceError: bazMessage","stack":"ServiceError: ServiceError: bazMessage\n    at Object.login (C:\\Projects\\ex_team\\services\\modules\\auth\\error-handler.js:8:19)\n    at process._tickCallback (internal/process/next_tick.js:68:7)"}
{"name":"ServiceError","date":"2019-08-21T06:05:39.812Z","status":301,"level":"error","message":"bazMessagebazMessage","stack":"ServiceError: bazMessage\n    at Object.login (C:\\Projects\\ex_team\\services\\modules\\auth\\error-handler.js:8:19)\n    at process._tickCallback (internal/process/next_tick.js:68:7)"}
{"name":"authError","date":"2019-08-21T06:10:13.825Z","status":401,"level":"error","message":"Invalid UserName or passwordInvalid UserName or password","stack":"authError: Invalid UserName or password\n    at Object.login (C:\\Projects\\ex_team\\services\\modules\\auth\\errorHandler.js:8:19)\n    at process._tickCallback (internal/process/next_tick.js:68:7)"}
{"level":"error","message":"Invalid UserName or passwordInvalid UserName or password","stack":"Error: Invalid UserName or password\n    at Object.login (C:\\Projects\\ex_team\\services\\modules\\auth\\errorHandler.js:8:19)\n    at process._tickCallback (internal/process/next_tick.js:68:7)"}
{"name":"authError","date":"2019-08-21T06:12:46.306Z","status":401,"level":"error","message":"Invalid UserName or passwordInvalid UserName or password","stack":"authError: Invalid UserName or password\n    at Object.login (C:\\Projects\\ex_team\\services\\modules\\auth\\errorHandler.js:8:19)\n    at process._tickCallback (internal/process/next_tick.js:68:7)"}
{"name":"authError","date":"2019-08-21T06:16:30.543Z","status":401,"level":"error","message":"Invalid UserName or passwordInvalid UserName or password","stack":"authError: Invalid UserName or password\n    at Object.login (C:\\Projects\\ex_team\\services\\modules\\auth\\errorHandler.js:8:19)\n    at process._tickCallback (internal/process/next_tick.js:68:7)"}
{"name":"authError","date":"2019-08-21T06:16:55.179Z","status":401,"level":"error","message":"Invalid UserName or passwordInvalid UserName or password","stack":"authError: Invalid UserName or password\n    at Object.login (C:\\Projects\\ex_team\\services\\modules\\auth\\errorHandler.js:8:19)\n    at process._tickCallback (internal/process/next_tick.js:68:7)"}
{"name":"authError","date":"2019-08-21T06:17:43.776Z","status":401,"level":"error","message":"Invalid UserName or passwordInvalid UserName or password","stack":"authError: Invalid UserName or password\n    at Object.login (C:\\Projects\\ex_team\\services\\modules\\auth\\errorHandler.js:8:19)\n    at process._tickCallback (internal/process/next_tick.js:68:7)"}
{"name":"authError","date":"2019-08-21T06:18:01.909Z","status":300,"level":"error","message":"Invalid UserName or passwordInvalid UserName or password","stack":"authError: Invalid UserName or password\n    at Object.login (C:\\Projects\\ex_team\\services\\modules\\auth\\errorHandler.js:8:19)\n    at process._tickCallback (internal/process/next_tick.js:68:7)"}
{"name":"authError","date":"2019-08-21T06:37:01.887Z","status":400,"level":"error","message":"Invalid UserName or passwordInvalid UserName or password","stack":"authError: Invalid UserName or password\n    at Object.login (C:\\Projects\\ex_team\\services\\modules\\auth\\errorHandler.js:8:19)\n    at process._tickCallback (internal/process/next_tick.js:68:7)"}
{"name":"authError","date":"2019-08-21T06:38:05.376Z","status":400,"level":"error","message":"\"userName\" is not allowed to be empty\"userName\" is not allowed to be empty","stack":"authError: \"userName\" is not allowed to be empty\n    at Object.login (C:\\Projects\\ex_team\\services\\modules\\auth\\errorHandler.js:8:19)"}
{"name":"authError","date":"2019-08-21T06:38:23.362Z","status":403,"level":"error","message":"\"userName\" is not allowed to be empty\"userName\" is not allowed to be empty","stack":"authError: \"userName\" is not allowed to be empty\n    at Object.login (C:\\Projects\\ex_team\\services\\modules\\auth\\errorHandler.js:8:19)"}
{"name":"authError","date":"2019-08-21T06:56:45.900Z","status":400,"level":"error","message":"password is not definedpassword is not defined","stack":"authError: password is not defined\n    at Object.login (C:\\Projects\\ex_team\\services\\modules\\auth\\errorHandler.js:8:19)"}
{"name":"authError","date":"2019-08-21T06:57:18.890Z","status":400,"level":"error","message":"uscerName is not defineduscerName is not defined","stack":"authError: uscerName is not defined\n    at Object.login (C:\\Projects\\ex_team\\services\\modules\\auth\\errorHandler.js:8:19)"}
{"name":"authError","date":"2019-08-21T06:58:12.432Z","status":400,"level":"error","message":"passdword is not definedpassdword is not defined","stack":"authError: passdword is not defined\n    at Object.login (C:\\Projects\\ex_team\\services\\modules\\auth\\errorHandler.js:8:19)"}
{"name":"authError","date":"2019-08-21T06:58:54.904Z","status":400,"level":"error","message":"passdword is not definedpassdword is not defined","stack":"authError: passdword is not defined\n    at Object.login (C:\\Projects\\ex_team\\services\\modules\\auth\\errorHandler.js:8:19)"}
{"level":"error","message":"res is not definedres is not defined","stack":"ReferenceError: res is not defined\n    at Object.login (C:\\Projects\\ex_team\\services\\modules\\auth\\errorHandler.js:11:17)"}
{"level":"error","message":"res is not definedres is not defined","stack":"ReferenceError: res is not defined\n    at Object.login (C:\\Projects\\ex_team\\services\\modules\\auth\\errorHandler.js:12:17)"}
{"level":"error","message":"res is not definedres is not defined","stack":"ReferenceError: res is not defined\n    at Object.login (C:\\Projects\\ex_team\\services\\modules\\auth\\errorHandler.js:9:17)\n    at process._tickCallback (internal/process/next_tick.js:68:7)"}
{"name":"authError","date":"2019-08-21T07:00:56.976Z","status":400,"level":"error","message":"Invalid UserName or passwordInvalid UserName or password","stack":"authError: Invalid UserName or password\n    at Object.login (C:\\Projects\\ex_team\\services\\modules\\auth\\errorHandler.js:9:23)\n    at process._tickCallback (internal/process/next_tick.js:68:7)"}
{"level":"error","message":"res is not definedres is not defined","stack":"ReferenceError: res is not defined\n    at Object.login (C:\\Projects\\ex_team\\services\\modules\\auth\\errorHandler.js:11:17)"}
{"name":"authError","date":"2019-08-21T07:04:46.217Z","status":400,"level":"error","message":"Invalid UserName or passwordInvalid UserName or password","stack":"authError: Invalid UserName or password\n    at Object.login (C:\\Projects\\ex_team\\services\\modules\\auth\\errorHandler.js:9:23)\n    at process._tickCallback (internal/process/next_tick.js:68:7)"}
{"level":"error","message":"res is not definedres is not defined","stack":"ReferenceError: res is not defined\n    at Object.login (C:\\Projects\\ex_team\\services\\modules\\auth\\errorHandler.js:11:17)"}
{"name":"authError","date":"2019-08-21T07:06:58.071Z","status":400,"level":"error","message":"Something went wrong. Please try again laterSomething went wrong. Please try again later","stack":"authError: Something went wrong. Please try again later\n    at Object.login (C:\\Projects\\ex_team\\services\\modules\\auth\\errorHandler.js:11:23)"}
{"name":"authError","date":"2019-08-21T07:11:50.585Z","status":400,"disc":"db.auth.getLoginBydUserName is not a function","level":"error","message":"Something went wrong. Please try again laterSomething went wrong. Please try again later","stack":"authError: Something went wrong. Please try again later\n    at Object.login (C:\\Projects\\ex_team\\services\\modules\\auth\\errorHandler.js:11:23)"}
{"name":"authError","date":"2019-08-21T07:14:11.720Z","status":400,"disc":"","level":"error","message":"Invalid UserName or passwordInvalid UserName or password","stack":"authError: Invalid UserName or password\n    at Object.login (C:\\Projects\\ex_team\\services\\modules\\auth\\errorHandler.js:9:23)\n    at process._tickCallback (internal/process/next_tick.js:68:7)"}
{"name":"authError","date":"2019-08-21T07:14:18.351Z","status":400,"disc":"passdword is not defined","level":"error","message":"Something went wrong. Please try again laterSomething went wrong. Please try again later","stack":"authError: Something went wrong. Please try again later\n    at Object.login (C:\\Projects\\ex_team\\services\\modules\\auth\\errorHandler.js:11:23)"}
{"name":"authError","date":"2019-08-21T07:16:10.334Z","status":400,"disc":"","level":"error","message":"Invalid UserName or passwordInvalid UserName or password","stack":"authError: Invalid UserName or password\n    at Object.login (C:\\Projects\\ex_team\\services\\modules\\auth\\errorHandler.js:9:23)\n    at process._tickCallback (internal/process/next_tick.js:68:7)"}
{"name":"authError","date":"2019-08-21T07:19:47.061Z","status":400,"description":"","level":"error","message":"\"password\" length must be at least 3 characters long\"password\" length must be at least 3 characters long","stack":"authError: \"password\" length must be at least 3 characters long\n    at Object.login (C:\\Projects\\ex_team\\services\\modules\\auth\\errorHandler.js:9:23)"}
{"name":"authError","date":"2019-08-21T07:19:51.352Z","status":400,"description":"","level":"error","message":"Invalid UserName or passwordInvalid UserName or password","stack":"authError: Invalid UserName or password\n    at Object.login (C:\\Projects\\ex_team\\services\\modules\\auth\\errorHandler.js:9:23)"}
{"name":"authError","date":"2019-08-21T07:40:28.226Z","status":400,"description":"","level":"error","message":"Invalid UserName or passwordInvalid UserName or password","stack":"authError: Invalid UserName or password\n    at Object.login (C:\\Projects\\ex_team\\services\\modules\\auth\\errorHandler.js:9:23)"}
{"level":"error","message":"auth.login is not a functionauth.login is not a function","stack":"TypeError: auth.login is not a function\n    at asyncError (C:\\Projects\\ex_team\\controller\\auth.js:21:30)"}
{"name":"authError","date":"2019-08-21T07:59:38.217Z","status":400,"description":"","level":"error","message":"\"password\" is required\"password\" is required","stack":"authError: \"password\" is required\n    at Object.login (C:\\Projects\\ex_team\\services\\modules\\auth\\errorHandler.js:9:23)"}
{"name":"customerError","date":"2019-08-21T08:10:02.600Z","status":200,"description":"db.customer.lisdtCustomerType is not a function","level":"error","message":"Something went wrong. Please try again laterSomething went wrong. Please try again later","stack":"customerError: Something went wrong. Please try again later\n    at Object.listCustomerType (C:\\Projects\\ex_team\\services\\modules\\customer\\errorHandler.js:12:23)"}
{"name":"customerError","date":"2019-08-21T08:14:18.723Z","status":500,"description":"db.customer.listCusstomerType is not a function","level":"error","message":"Something went wrong. Please try again laterSomething went wrong. Please try again later","stack":"customerError: Something went wrong. Please try again later\n    at Object.listCustomerType (C:\\Projects\\ex_team\\services\\modules\\customer\\errorHandler.js:12:23)"}
{"name":"user","date":"2019-08-21T08:31:13.338Z","status":400,"description":"cusdtomerID is not defined","level":"error","message":"Something went wrong. Please try again laterSomething went wrong. Please try again later","stack":"user: Something went wrong. Please try again later\n    at Object.getPersonById (C:\\Projects\\ex_team\\services\\modules\\user\\errorHandler.js:12:23)"}
{"name":"user","date":"2019-08-21T08:31:20.820Z","status":400,"description":"cusdtomerID is not defined","level":"error","message":"Something went wrong. Please try again laterSomething went wrong. Please try again later","stack":"user: Something went wrong. Please try again later\n    at Object.getPersonById (C:\\Projects\\ex_team\\services\\modules\\user\\errorHandler.js:12:23)"}
{"name":"authError","date":"2019-08-21T19:58:40.026Z","status":400,"description":"","level":"error","message":"\"userName\" is not allowed to be empty\"userName\" is not allowed to be empty","stack":"authError: \"userName\" is not allowed to be empty\n    at Object.login (C:\\Projects\\ex_team\\services\\modules\\auth\\errorHandler.js:10:23)"}
{"name":"authError","date":"2019-08-21T20:01:33.177Z","status":400,"description":"","level":"error","message":"\"userName\" is not allowed to be empty\"userName\" is not allowed to be empty","stack":"authError: \"userName\" is not allowed to be empty\n    at Object.login (C:\\Projects\\ex_team\\services\\modules\\auth\\errorHandler.js:10:23)"}
{"name":"authError","date":"2019-08-21T20:04:33.920Z","status":400,"description":"","level":"error","message":"\"password\" length must be at least 3 characters long\"password\" length must be at least 3 characters long","stack":"authError: \"password\" length must be at least 3 characters long\n    at Object.login (C:\\Projects\\ex_team\\services\\modules\\auth\\errorHandler.js:10:23)"}
{"name":"authError","date":"2019-08-21T20:11:41.021Z","status":400,"description":"","level":"error","message":"\"password\" length must be at least 3 characters long\"password\" length must be at least 3 characters long","stack":"authError: \"password\" length must be at least 3 characters long\n    at Object.login (C:\\Projects\\ex_team\\services\\modules\\auth\\errorHandler.js:10:23)"}
{"name":"authError","date":"2019-08-21T20:11:49.380Z","status":400,"description":"","level":"error","message":"\"userName\" length must be at least 3 characters long\"userName\" length must be at least 3 characters long","stack":"authError: \"userName\" length must be at least 3 characters long\n    at Object.login (C:\\Projects\\ex_team\\services\\modules\\auth\\errorHandler.js:10:23)"}
{"level":"error","message":["error","data"]}
{"code":"ELOGIN","originalError":{"message":"Login failed for user 'sa1'.","code":"ELOGIN"},"name":"ConnectionError","level":"error","message":"error,dataLogin failed for user 'sa1'.","stack":"ConnectionError: Login failed for user 'sa1'.\n    at Connection.tedious.once.err (C:\\Projects\\ex_team\\node_modules\\mssql\\lib\\tedious.js:244:17)\n    at Object.onceWrapper (events.js:277:13)\n    at Connection.emit (events.js:189:13)\n    at Connection.processLogin7Response (C:\\Projects\\ex_team\\node_modules\\tedious\\lib\\connection.js:1397:14)\n    at Connection.message (C:\\Projects\\ex_team\\node_modules\\tedious\\lib\\connection.js:1932:14)\n    at Connection.dispatchEvent (C:\\Projects\\ex_team\\node_modules\\tedious\\lib\\connection.js:1084:36)\n    at MessageIO.messageIo.on (C:\\Projects\\ex_team\\node_modules\\tedious\\lib\\connection.js:984:14)\n    at MessageIO.emit (events.js:189:13)\n    at Message.message.on (C:\\Projects\\ex_team\\node_modules\\tedious\\lib\\message-io.js:32:14)\n    at Message.emit (events.js:194:15)"}
{"level":"error","message":["error","data"]}
{"level":"error","message":"error,dataCannot read property 'close' of null","stack":"TypeError: Cannot read property 'close' of null\n    at closePool (C:\\Projects\\ex_team\\repository\\index.js:15:23)\n    at sql.ConnectionPool.connect.then.catch (C:\\Projects\\ex_team\\repository\\index.js:50:23)\n    at process._tickCallback (internal/process/next_tick.js:68:7)"}
{"name":"customerError","date":"2019-08-21T20:20:23.510Z","status":500,"description":"","level":"error","message":"Something went wrong. Please try again laterSomething went wrong. Please try again later","stack":"customerError: Something went wrong. Please try again later\n    at Object.listCustomerType (C:\\Projects\\ex_team\\services\\modules\\customer\\errorHandler.js:12:23)\n    at process._tickCallback (internal/process/next_tick.js:68:7)"}
{"level":"error","message":["error","data"]}
{"code":"ELOGIN","originalError":{"message":"Login failed for user 'sa1'.","code":"ELOGIN"},"name":"ConnectionError","level":"error","message":"error,dataLogin failed for user 'sa1'.","stack":"ConnectionError: Login failed for user 'sa1'.\n    at Connection.tedious.once.err (C:\\Projects\\ex_team\\node_modules\\mssql\\lib\\tedious.js:244:17)\n    at Object.onceWrapper (events.js:277:13)\n    at Connection.emit (events.js:189:13)\n    at Connection.processLogin7Response (C:\\Projects\\ex_team\\node_modules\\tedious\\lib\\connection.js:1397:14)\n    at Connection.message (C:\\Projects\\ex_team\\node_modules\\tedious\\lib\\connection.js:1932:14)\n    at Connection.dispatchEvent (C:\\Projects\\ex_team\\node_modules\\tedious\\lib\\connection.js:1084:36)\n    at MessageIO.messageIo.on (C:\\Projects\\ex_team\\node_modules\\tedious\\lib\\connection.js:984:14)\n    at MessageIO.emit (events.js:189:13)\n    at Message.message.on (C:\\Projects\\ex_team\\node_modules\\tedious\\lib\\message-io.js:32:14)\n    at Message.emit (events.js:194:15)"}
{"level":"error","message":["error","data"]}
{"level":"error","message":"error,dataCannot read property 'close' of null","stack":"TypeError: Cannot read property 'close' of null\n    at closePool (C:\\Projects\\ex_team\\repository\\index.js:15:23)\n    at sql.ConnectionPool.connect.then.catch (C:\\Projects\\ex_team\\repository\\index.js:50:23)\n    at process._tickCallback (internal/process/next_tick.js:68:7)"}
{"name":"customerError","date":"2019-08-21T20:22:35.813Z","status":500,"description":"","level":"error","message":"Something went wrong. Please try again laterSomething went wrong. Please try again later","stack":"customerError: Something went wrong. Please try again later\n    at Object.listCustomerType (C:\\Projects\\ex_team\\services\\modules\\customer\\errorHandler.js:12:23)\n    at process._tickCallback (internal/process/next_tick.js:68:7)"}
{"level":"error","message":["error","data"]}
{"code":"ELOGIN","originalError":{"message":"Login failed for user 'sa1'.","code":"ELOGIN"},"name":"ConnectionError","level":"error","message":"error,dataLogin failed for user 'sa1'.","stack":"ConnectionError: Login failed for user 'sa1'.\n    at Connection.tedious.once.err (C:\\Projects\\ex_team\\node_modules\\mssql\\lib\\tedious.js:244:17)\n    at Object.onceWrapper (events.js:277:13)\n    at Connection.emit (events.js:189:13)\n    at Connection.processLogin7Response (C:\\Projects\\ex_team\\node_modules\\tedious\\lib\\connection.js:1397:14)\n    at Connection.message (C:\\Projects\\ex_team\\node_modules\\tedious\\lib\\connection.js:1932:14)\n    at Connection.dispatchEvent (C:\\Projects\\ex_team\\node_modules\\tedious\\lib\\connection.js:1084:36)\n    at MessageIO.messageIo.on (C:\\Projects\\ex_team\\node_modules\\tedious\\lib\\connection.js:984:14)\n    at MessageIO.emit (events.js:189:13)\n    at Message.message.on (C:\\Projects\\ex_team\\node_modules\\tedious\\lib\\message-io.js:32:14)\n    at Message.emit (events.js:194:15)"}
{"level":"error","message":["error","data"]}
{"level":"error","message":"error,dataCannot read property 'close' of null","stack":"TypeError: Cannot read property 'close' of null\n    at closePool (C:\\Projects\\ex_team\\repository\\index.js:15:23)\n    at sql.ConnectionPool.connect.then.catch (C:\\Projects\\ex_team\\repository\\index.js:50:23)\n    at process._tickCallback (internal/process/next_tick.js:68:7)"}
{"name":"customerError","date":"2019-08-21T20:22:38.285Z","status":500,"description":"","level":"error","message":"Something went wrong. Please try again laterSomething went wrong. Please try again later","stack":"customerError: Something went wrong. Please try again later\n    at Object.listCustomerType (C:\\Projects\\ex_team\\services\\modules\\customer\\errorHandler.js:12:23)\n    at process._tickCallback (internal/process/next_tick.js:68:7)"}
{"level":"error","message":["error","data"]}
{"code":"ELOGIN","originalError":{"message":"Login failed for user 'sa1'.","code":"ELOGIN"},"name":"ConnectionError","level":"error","message":"error,dataLogin failed for user 'sa1'.","stack":"ConnectionError: Login failed for user 'sa1'.\n    at Connection.tedious.once.err (C:\\Projects\\ex_team\\node_modules\\mssql\\lib\\tedious.js:244:17)\n    at Object.onceWrapper (events.js:277:13)\n    at Connection.emit (events.js:189:13)\n    at Connection.processLogin7Response (C:\\Projects\\ex_team\\node_modules\\tedious\\lib\\connection.js:1397:14)\n    at Connection.message (C:\\Projects\\ex_team\\node_modules\\tedious\\lib\\connection.js:1932:14)\n    at Connection.dispatchEvent (C:\\Projects\\ex_team\\node_modules\\tedious\\lib\\connection.js:1084:36)\n    at MessageIO.messageIo.on (C:\\Projects\\ex_team\\node_modules\\tedious\\lib\\connection.js:984:14)\n    at MessageIO.emit (events.js:189:13)\n    at Message.message.on (C:\\Projects\\ex_team\\node_modules\\tedious\\lib\\message-io.js:32:14)\n    at Message.emit (events.js:194:15)"}
{"level":"error","message":["error","data"]}
{"level":"error","message":"error,dataCannot read property 'close' of null","stack":"TypeError: Cannot read property 'close' of null\n    at closePool (C:\\Projects\\ex_team\\repository\\index.js:15:23)\n    at sql.ConnectionPool.connect.then.catch (C:\\Projects\\ex_team\\repository\\index.js:50:23)\n    at process._tickCallback (internal/process/next_tick.js:68:7)"}
{"name":"authError","date":"2019-08-21T20:22:51.273Z","status":400,"description":"","level":"error","message":"Something went wrong. Please try again laterSomething went wrong. Please try again later","stack":"authError: Something went wrong. Please try again later\n    at Object.login (C:\\Projects\\ex_team\\services\\modules\\auth\\errorHandler.js:12:23)\n    at process._tickCallback (internal/process/next_tick.js:68:7)"}
{"message":"connection pool error","level":"error"}
{"code":"ELOGIN","originalError":{"message":"Login failed for user 'sa1'.","code":"ELOGIN"},"name":"ConnectionError","level":"error"}
{"message":"closePool error","level":"error"}
{"level":"error"}
{"name":"customerError","date":"2019-08-21T20:23:40.358Z","status":500,"description":"","level":"error","message":"Something went wrong. Please try again laterSomething went wrong. Please try again later","stack":"customerError: Something went wrong. Please try again later\n    at Object.listCustomerType (C:\\Projects\\ex_team\\services\\modules\\customer\\errorHandler.js:12:23)\n    at process._tickCallback (internal/process/next_tick.js:68:7)"}
{"name":"authError","date":"2019-08-21T20:53:19.881Z","status":400,"description":"","level":"error","message":"\"userName\" is not allowed to be empty\"userName\" is not allowed to be empty","stack":"authError: \"userName\" is not allowed to be empty\n    at Object.login (C:\\Projects\\ex_team\\services\\modules\\auth\\errorHandler.js:10:23)"}
{"name":"authError","date":"2019-08-21T20:54:23.654Z","status":400,"description":"","level":"error","message":"\"userName\" is not allowed to be empty\"userName\" is not allowed to be empty","stack":"authError: \"userName\" is not allowed to be empty\n    at Object.login (C:\\Projects\\ex_team\\services\\modules\\auth\\errorHandler.js:10:23)"}
{"message":"connection pool error","level":"error"}
{"code":"ELOGIN","originalError":{"message":"Login failed for user 'sa1'.","code":"ELOGIN"},"name":"ConnectionError","level":"error"}
{"message":"closePool error","level":"error"}
{"level":"error"}
{"name":"authError","date":"2019-08-21T20:54:49.723Z","status":400,"description":"","level":"error","message":"Something went wrong. Please try again laterSomething went wrong. Please try again later","stack":"authError: Something went wrong. Please try again later\n    at Object.login (C:\\Projects\\ex_team\\services\\modules\\auth\\errorHandler.js:12:23)\n    at process._tickCallback (internal/process/next_tick.js:68:7)"}
{"message":"connection pool error","level":"error"}
{"code":"ELOGIN","originalError":{"message":"Login failed for user 'sa1'.","code":"ELOGIN"},"name":"ConnectionError","level":"error"}
{"message":"closePool error","level":"error"}
{"level":"error"}
{"name":"authError","date":"2019-08-21T20:55:27.507Z","status":500,"description":"","level":"error","message":"Something went wrong. Please try again laterSomething went wrong. Please try again later","stack":"authError: Something went wrong. Please try again later\n    at Object.login (C:\\Projects\\ex_team\\services\\modules\\auth\\errorHandler.js:12:23)\n    at process._tickCallback (internal/process/next_tick.js:68:7)"}
